codeReview.js

```diff
@@ -0,0 +1,78 @@
+import { Router, Request, Response } from 'api-framework'; // express-like package
+ 
+   import { getDBConnection } from '../config/db';
+   import { User } from '../types/user';
+   import { sendEmail } from '../util/send_email';
+   import { sendText } from '../util/send_text';
+ 
+   const router = Router();
+   const db = getDBConnection();
+ 
+   const fetchUsersByEmail = async (emails: string[]) => {
```

Comment:
why use anonymous function, it changes the functionality of "this", I do not see this variable passed into any functions